marketplace_builder/views/pages/how-to/e-commerce/configuring-data-schema.liquid

```diff
@@ -0,0 +1,225 @@
+        width: 250
+        height: 345
+        transform: resize_to_fill
+      mini:
+        width: 500
+        height: 800
+        transform: resize_to_fit
+      normal:
+        width: 1500
+        height: 1200
+        transform: resize_to_fit
+```
+
+`photo` model will be using special field type called, well, `photo`. It helps automating upload and versions creation. You can [read more on `Custom Images`](/reference/custom-attributes/custom-images).
+
+```yaml
+# custom_model_types/stock_level.yml
+name: stock_level
+custom_attributes:
+  - name: product_id
+    attribute_type: string
+  - name: variant
+    attribute_type: string
+  - name: amount
+    attribute_type: int
+```
+
+## User
+
+`User` is a built-in data type in PlatformOS. It has some predefined fields already `name` and `email`. We can extend `User` with additional custom attributes by using user profiles. We will use `default` user profile to add a phone number field.
+
+```yaml
+# user_profile_types/default.yml
+name: default
+custom_attributes:
+  - name: phone
+    attribute_type: string
+  - name: admin
+    attribute_type: boolean
+```
+
+Every user will also be able to save multiple `addresses` to his contact book.
+
+```yaml
+name: address
+custom_attributes:
+  - name: label
+    attribute_type: string
+  - name: location
+    attribute_type: string
+  - name: default_address
+    attribute_type: boolean
+```
+
+{% capture content %}
+As you’ve probably noticed we’ve omitted `user_id` attribute. Every `Custom Model Type` in PlatformOS can be attached to another object via built-in `customizable_type` and `customizable_id` fields. We will be setting these values when creating forms.
+{% endcapture %}
+{% include 'alert/note', content: content %}
+
+{% capture content %}
+PlatformOS has a special field type called `CustomAddress`. It allows geolocation and detailed validation of the address. We will not be using that in this example, but go ahead and [read more on `Custom Address` field](/reference/custom-attributes/custom-addresses) on our documentation site.
+{% endcapture %}
+{% include 'alert/note', content: content %}
+
+## Order
+
+Last part of our shop data schema is related to `order`s created by clients. Every order can have multiple products attached and a `shipping_option` selected during checkout. Order is finished with `payment`, we will store history of all payments for given order (usually it will be just one, but we want to log every interaction with payments API as separate entry). Finally, we will store `order_history` describing any changes to order state (received, paid, shipped, etc.)
+
+```yaml
+# custom_model_types/order.yml
+name: order
+custom_attributes:
+  - name: billing_address
+    attribute_type: text
+  - name: shipping_address
+    attribute_type: text
+  - name: shipping_option_id
+    attribute_type: string
+  - name: shipping_option_name
+    attribute_type: string
+  - name: shipping_price
+    attribute_type: int
```

Comment:
Since integer works and we used it all the way so far everywhere i suggest keeping it explicit

```suggestion
    attribute_type: integer
```